pyproject.toml

```diff
@@ -55,8 +55,8 @@ ignore_patterns = [
 ]
 
 [tool.ruff]
-# Allow lines to be as long as 80.
-line-length = 80
+# Allow lines to be as long as 88 to be compatible with Black.
+line-length = 88
```

Comment:
Ruff has drop-in parity with black, so 88 is the default https://docs.astral.sh/ruff/configuration/

```suggestion
```